app/src/main/java/com/example/concordia_campus_guide/Models/Coordinates.java

```diff
@@ -45,4 +45,18 @@ public double getLongitude() {
     public void setLongitude(double longitude) {
         this.longitude = longitude;
     }
+
+    /**
+     * This method returns the euclidean distance between this and another point based on the pythagorean theorem.
+     * @param other
+     * @return
+     */
+    public double getEuclideanDistanceFrom(Coordinates other){
+        final double resultLatDiff = Math.abs(this.toListDouble().get(0)
```

Comment:
any reason why we aren't using the longitude and latitude directly instead of the toListDouble?

Reply:
Nope. Will change it.